src/shared/cryptsetup-util.h

```diff
@@ -37,6 +37,12 @@ extern int (*sym_crypt_set_pbkdf_type)(struct crypt_device *cd, const struct cry
 extern int (*sym_crypt_token_json_get)(struct crypt_device *cd, int token, const char **json);
 extern int (*sym_crypt_token_json_set)(struct crypt_device *cd, int token, const char *json);
 extern int (*sym_crypt_volume_key_get)(struct crypt_device *cd, int keyslot, char *volume_key, size_t *volume_key_size, const char *passphrase, size_t passphrase_size);
+#if HAVE_CRYPT_TOKEN_MAX
+extern int (*sym_crypt_token_max)(const char *type);
+#else
+/* As a fallback, use the same hard-coded value libcryptsetup uses internally. */
+static inline int sym_crypt_token_max(_unused_ const char *type) { return 32; }
```

Comment:
i don't see no need to cram this all in the same line... i.e. please add line breaks like we would for any other function...

also, maybe add `assert(streq(type, CRYPT_LUKS2));` as first line of the function in order to encode our expectation here.

Reply:
sure thing, done